packages/orchestration/test/assets.fixture.ts

```diff
@@ -8,7 +8,7 @@ export const assets = {
         'ATOM is the native cryptocurrency of the Cosmos network, designed to facilitate interoperability between multiple blockchains through its innovative hub-and-spoke model.',
       extended_description:
         "ATOM, the native cryptocurrency of the Cosmos network, is essential for achieving the project's goal of creating an 'Internet of Blockchains.' Launched in 2019, Cosmos aims to solve the scalability, usability, and interoperability issues prevalent in existing blockchain ecosystems. The Cosmos Hub, the central blockchain of the network, uses ATOM for transaction fees, staking, and governance. By staking ATOM, users can earn rewards and participate in governance, influencing decisions on network upgrades and changes.\n\nCosmos leverages the Tendermint consensus algorithm to achieve high transaction throughput and fast finality. Its Inter-Blockchain Communication (IBC) protocol enables seamless data and value transfer between different blockchains, fostering a highly interconnected and collaborative ecosystem. The flexibility and scalability offered by Cosmos have attracted numerous projects, enhancing its utility and adoption. ATOM's role in securing the network and facilitating governance underscores its importance in the broader blockchain landscape.",
-      denom_units: [
+      denomUnits: [
```

Comment:
This was sourced from the chain registry. I don't think it's cost-effective to be different.

https://github.com/cosmos/chain-registry/blob/89fe1c12a085b7b128a181118c9600e781363168/cosmoshub/assetlist.json#L8

Reply:
never mind.

looks like we're already tweaking such names.

```js
      portId: channelFrom.port_id,
```
https://github.com/Agoric/agoric-sdk/blob/master/packages/orchestration/src/utils/registry.js#L68C1-L68C35